spec/policies.yml

```
@@ -102,7 +102,7 @@ components:
           ##### Permissions required
 
           `create` privilege on the policy."
-        operationId: "updatePolicy"
+        operationId: "append"
```

```suggestion
        operationId: "load"
```
it should still use `load` instead of `append`

---

@telday this is still open